src/style.css

```
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.searchBars {
+  display: flex;
+  margin: 20px 0;
+  position: relative;
+  background: #fff;
+}
+
+.bar {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.big {
+  font-size: 16px;
+  width: 100%;
+  height: 68px;
+  padding: 0 62px;
```

use shorthand properties when you have padding/margin from one side
```suggestion
  padding-left: 62px;
```